### PR TITLE
Add prepend argument to runtests.py verifier

### PR DESCRIPTION
**Pull request by Ch3LL**
For MacOSX tests we need to set the LANG before running the command. This allows us to ensure this command is run on macosx:

`LANG='en_US.UTF-8' python3 /tmp/kitchen/testing/tests/runtests.py....`

with the configuration:

```
            verifier:
              prepend: LANG='en_US.UTF-8'
```

### CODE REVIEW

File: lib/kitchen/verifier/runtests.rb [hunk 1]
```diff
@@ -35,6 +36,7 @@ def call(state)
 	  config[:from_filenames] = repo.diff("origin/#{ENV['CHANGE_TARGET']}", "origin/#{ENV['BRANCH_NAME']}").name_status.keys.select{|file| file.end_with?('.py')}
         end
         command = [
+          (config[:prepend]),
```
Comment (s0undt3ch) [inline lib/kitchen/verifier/runtests.rb]:
```
(config[:prepend] ? "#{config[:prepend]}" : ''),
```

File: lib/kitchen/verifier/runtests.rb [hunk 2]
```diff
@@ -22,6 +22,7 @@ class Runtests < Kitchen::Verifier::Base
       default_config :windows, false
       default_config :enable_filenames, false
       default_config :from_filenames, []
+      default_config :prepend, ''
```
Comment (s0undt3ch) [inline lib/kitchen/verifier/runtests.rb]:
```
default_config :prepend, false
```

**Comment by s0undt3ch** [comment]
Something like the comments is better

**Comment by Ch3LL** [comment]
@s0undt3ch done thanks for the review

**Pull request merged**